src/aspire/denoising/denoised_src.py

```diff
@@ -1,31 +1,29 @@
 import logging
 
-import numpy as np
-
-from aspire.image import Image
+from aspire.denoising import Denoiser
 from aspire.source import ImageSource
 
 logger = logging.getLogger(__name__)
 
 
 class DenoisedImageSource(ImageSource):
```

Comment:
Maybe `DenoisedSource` (cf. `OrientedSource`, `RelionSource`, `ClassAvgSource` etc. where we skip `Image`).

Reply:
Sure.